### PR TITLE
✍🏽 Renaming Professors

### PR DESCRIPTION
**Pull request by Kanishk-K**
Rename "Unknown Professor" to "Unknown Instructor"
Obfuscate Anonymous Professor
Reattribute Challou's grade information to Dan Challou.

Regen RMP?

### CODE REVIEW

File: data-app/dist_gen.py [hunk 1]
```diff
@@ -219,8 +219,8 @@ def fetch_asr(dept_dist:DepartmentDistribution,term:int) -> None:
     else:
         print("No new professors found.")
     
-    if session.query(Professor).filter(Professor.name == "Unknown Professor").first() == None:
-        session.add(Professor(name="Unknown Professor"))
+    if session.query(Professor).filter(Professor.name == "Unknown Instructor").first() == None:
+        session.add(Professor(name="Unknown Instructor"))
         session.commit()
 
     print("Finished Prof Insertion")
```
Comment (JosephMcIndoo) [inline data-app/dist_gen.py:226]: Finished Instructor Insertion

File: data-app/dist_gen.py [hunk 2]
```diff
@@ -219,8 +219,8 @@ def fetch_asr(dept_dist:DepartmentDistribution,term:int) -> None:
     else:
         print("No new professors found.")
```
Comment (JosephMcIndoo) [inline data-app/dist_gen.py:220]: No new instructor found

File: data-app/dist_gen.py (hunk 1 above)
Comment (JosephMcIndoo) [inline data-app/dist_gen.py:226]: Finished instructor insertion

File: data-app/dist_gen.py [hunk 3]
```diff
@@ -208,7 +208,7 @@ def fetch_asr(dept_dist:DepartmentDistribution,term:int) -> None:
     df = pd.read_csv("CLASS_DATA/combined_clean_data.csv",dtype={"CLASS_SECTION":str})
     print("Loaded Data!")
     print("Adding Profs")
```
Comment (JosephMcIndoo) [inline data-app/dist_gen.py:210]: Adding Instructors

**Review by JosephMcIndoo — APPROVED** [review]
Probably some more minor prof -> instructor changes that need to be done in the print statements. Not that important though.

**Review by JosephMcIndoo — APPROVED** [review]
Probably some more minor prof -> instructor changes that need to be done in the print statements. Not that important though.

**Comment by Kanishk-K** [comment]
> Probably some more minor prof -> instructor changes that need to be done in the print statements. Not that important though.

Noticed this as well, mostly wanted to work on database-affecting code for now. The print statements, while important, can wait. I'm also inclined to rename the SQL table and object but that's a little more complicated and I'll have to create a migration for that. I'll probably do something after Spring 2023 data is inserted so I'll keep the issue open for now.

**Pull request merged**